docs/source/.ipynb_checkpoints/evaluation-checkpoint.ipynb

```diff
@@ -0,0 +1,10508 @@
+{
```

Comment:
Please add ipynb checkpoints to the gitignore.

Reply:
Ah, I'm confused. Is this a checkpoint or the notebook itself?

Reply:
Sorry, added checkpoint to gitignore.